fedimint-server/src/consensus/server.rs

```diff
@@ -328,12 +328,12 @@ impl ConsensusServer {
     pub async fn run_session(&self, session_index: u64) -> anyhow::Result<()> {
         // if all nodes are correct the session will take 45 to 60 seconds. The
         // more nodes go offline the longer the session will take to complete.
-        const EXPECTED_ROUNDS_PER_SESSION: usize = 45 * 4;
+        const EXPECTED_ROUNDS_PER_SESSION: usize = 45 * 20;
         // this constant needs to be 3000 or less to guarantee that the session
         // can never reach MAX_ROUNDs.
         const EXPONENTIAL_SLOWDOWN_OFFSET: usize = 3 * EXPECTED_ROUNDS_PER_SESSION;
```

Comment:
We are getting close to this limit, is this a concern @joschisan? I tried to up `MAX_ROUND` to 10k to have more safety margin again, but that lead to unclear errors.

Reply:
Getting close to the limit is not a concern. No safety margin necessary here.